src/routes/middlewares/validateUser.js

```diff
@@ -0,0 +1,43 @@
+import Joi from 'joi';
+import { errorResponse, verifyToken } from '../utils/helpers';
+import { newUserSchema } from '../../utils/validationSchema';
+
+/**
+ * Input validator for a new user account
+ * @param {Object} req - Express HTTP Request Object
+ * @param {Object} res - Express HTTP Response Object
+ * @param {Object} next - On to the next handler
+ * @returns {Object} Error Response if validation fails
+ * @callback next when validation passes
+ */
+const validateNewUser = (req, res, next) => {
```

Comment:
Am sure we have this function somewhere in the codebase, why create new one here?

Reply:
Also, I don't see where you are using this in this PR.

Reply:
It came back as an old file accidentally. It is now deleted completely.